package.json

```diff
@@ -5,6 +5,8 @@
   "productName": "Git-it",
   "main": "main.js",
   "scripts": {
+    "app": "electron .",
```

Comment:
this can be `"start": "electron ."` because start is special :sparkles: and we can just do `npm start` rather than `npm run app`. Or we can also just add it on another line if you like the `run` bit.